source/index.html.md

```diff
@@ -293,14 +345,75 @@ Definition of the object attributes
 | files.billOfLading.container.containerNo         ||
 | files.billOfLading.container.containerType         ||
 | files.billOfLading.container.seals         ||
+| files.billOfLading.packLine.hsCode   ||
+| files.billOfLading.packLine.containerNo   ||
+| files.billOfLading.packLine.goodsDescription   ||
+| files.billOfLading.packLine.isGoodsSegment   ||
+| files.billOfLading.packLine.marksAndNumbers   ||
+| files.billOfLading.packLine.numberPieces   ||
+| files.billOfLading.packLine.pieceType   ||
```

Comment:
1) can we call this Package Type
2) set the description - The package's type. For list of possible values, see [List of PackageType values]